apps/ui/src/networks/offchain/actions.ts

```diff
@@ -393,7 +409,23 @@ export function createActions(
         data: { network: networkId, space: spaceId, ...(from ? { from } : {}) }
       });
     },
-    setAlias(web3: Web3Provider, alias: string) {
+    async setAlias(web3: any, alias: string) {
+      const web3ChainId =
+        web3.provider?.chainId || web3.provider?.provider?.chainId;
+
+      if (STARKNET_CHAIN_IDS.includes(web3ChainId)) {
+        await verifyStarknetNetwork(
+          web3,
+          // Argent X is returning SN_... while Braavos is returning Ox....
+          starknetConstants.StarknetChainId[web3ChainId] || web3ChainId
```

Comment:
This doesn't work for me on Argent X. If I'm on sepolia and try to follow it won't change my wallet network because it expects us to pass `SN_MAIN` and we pass hex ID.

Reply:
Should be fixed now.